entity-service/src/main/kotlin/com/egm/stellio/entity/repository/StandaloneNeo4jSearchRepository.kt

```diff
@@ -0,0 +1,79 @@
+package com.egm.stellio.entity.repository
+
+import com.egm.stellio.entity.util.*
+import com.egm.stellio.entity.util.QueryUtils.buildCypherQueryToQueryEntities
+import com.egm.stellio.shared.util.toUri
+import org.neo4j.ogm.session.Session
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty
+import org.springframework.stereotype.Component
+import java.net.URI
+import java.util.regex.Pattern
+
+@Component
+@ConditionalOnProperty("application.authentication.enabled", havingValue = "false")
+class StandaloneNeo4jSearchRepository(
+    private val session: Session
+) : SearchRepository {
+
+    override fun getEntities(params: Map<String, Any?>, userId: String, page: Int, limit: Int): Pair<Int, List<URI>> {
```

Comment:
the two implementations (with ou without authentication enabled) are finally very similar, no?

Reply:
Yes, but not for all clauses.